inframanager/config.yaml

```diff
@@ -23,12 +23,13 @@ Infrap4dGnmiServer:
 #      client-key: "key_path"
 #      ca-cert: "ca_path"
 #    If not specified, the default values are
-#      client-cert: "/tmp/inframanager/client/tls.crt"
-#      client-key: "/tmp/inframanager/client/tls.key"
-#      ca-cert: "/tmp/inframanager/client/ca.crt"
+#      client-cert: "/etc/pki/inframanager/client/tls.crt"
+#      client-key: "/etc/pki/inframanager/client/tls.key"
+#      ca-cert: "/etc/pki/inframanager/client/ca.crt"
   conn: mtls
 
 InfraManager: 
+  addr: 127.0.0.1:50002
```

Comment:
Don't we need the same for InfraAgent as well?

Reply:
The inframanager server will be running on the acc whereas the agent and calico will be on the host. So, the agent doesn't need a configurable ip address as of now.

Reply:
So, after these changes, who uses
InfraAgentAddr and InfraAgentPort
configurations from types.go file?